Update azure-security-disk-encryption-tsg.md

ejarvi opened the pull request:
Add link to KB article for cases where encryption status is displayed incorrectly on Azure Management Portal

azurecla commented:
Hi __@LuisCabrer__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (luisca). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

ejarvi committed:
Update azure-security-disk-encryption-tsg.md

Add link to KB article for cases where encryption status is displayed incorrectly on Azure Management Portal
